src/Advertising.js

```diff
@@ -266,6 +266,25 @@ export default class Advertising {
     }
   }
 
+  defineInterstitialSlot() {
+    if (this.config.interstitialSlot) {
```

Comment:
Please add the new `interstitialSlot` config option to the [AdvertisingConfigPropType](https://github.com/eBayClassifiedsGroup/react-advertising/blob/master/src/components/utils/AdvertisingConfigPropType.js)